INSTALL.md

```diff
@@ -25,7 +25,9 @@ Linux
 
 Use `make dependencies` to map the native libraries to your system, fetch the remaining CLI dependencies using [NuGet](http://www.nuget.org) and place them at the appropriate places.
 
-To compile OpenRA, run `make all` from the command line.
+To checkout the latest stable for online play run run `git checkout master`
```

Comment:
There's nothing wrong with master.  `make version` will pick up the tag name itself.